README.md

```diff
@@ -128,10 +137,12 @@ docker run -d --name grafana \
       grafana/grafana:7.5.4
 ```
 
-You can now start your NuoDB database with the NuoDB Collector.
-For more info on how to start your database, please refer to the [NuoDB Docker Blog Part I](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i) and the readme for [NuoDB Collector](https://github.com/nuodb/nuodb-collector).
+4. If you haven't already, [start your NuoDB database](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i).
```

Comment:
Isn't it better to point to our official doc - https://doc.nuodb.com/nuodb/latest/quick-start-guide/docker/?

Reply:
Thank you for your suggestion. I will replace the link.